client/app/controllers/collection/group/index.js

```diff
@@ -15,10 +15,23 @@ export default Ember.Controller.extend({
               this.set('showDeleteItemConfirmation', false);
           },
         deletePartial(){
-            // for the moment this will do the same as deletefull.
+            let collection = this.get('model.collection');
+            let items = this.get('model.items');
+            items.forEach(item => {
+                item.set('group', null);
+                item.save();
+            });
+            this.get('model').destroyRecord().then(() =>
+              this.transitionToRoute('collection', collection)
+            );
+            this.send('clearModals');
         },
         deleteFull(){
-
+            let collection = this.get('model.collection');
+            this.get('model').destroyRecord().then(() =>
+              this.transitionToRoute('collection', collection)
```

Comment:
Do the items get deleted when this happens? Because of the ember models it should right?